src/System.Device.Gpio/System/Device/Gpio/Drivers/SysFsDriver.Linux.cs

```diff
@@ -15,13 +15,33 @@ namespace System.Device.Gpio.Drivers
     public class SysFsDriver : UnixDriver
     {
         private const string GpioBasePath = "/sys/class/gpio";
+        private const string GpioChip = "gpiochip";
+        private const string GpioLabel = "/label";
+        private const string GpioContoller = "pinctrl";
+        private const string GpioOffsetBase = "/base";
         private int _pollFileDescriptor = -1;
         private Thread _eventDetectionThread;
         private int _pinsToDetectEventsCount;
         private readonly static CancellationTokenSource s_eventThreadCancellationTokenSource = new CancellationTokenSource();
         private readonly List<int> _exportedPins = new List<int>();
         private readonly Dictionary<int, UnixDriverDevicePin> _devicePins = new Dictionary<int, UnixDriverDevicePin>();
         private readonly int _pollingTimeoutInMilliseconds = Convert.ToInt32(TimeSpan.FromMilliseconds(1).TotalMilliseconds);
+        private int _pinOffset = 0;
```

Comment:
>= 0 [](start = 31, length = 3)

I don't think we need to have this explicit initialization to 0.

Reply:
Could you share the reason? 
For me if no any offset number found it should be 0 as this number is added to pin number for any further functionality.

Reply:
ints are value types, so they will always get initialized with 0.

Reply:
yeah I know that, just didn't think from that side 😛